install/installer/pkg/components/spicedb/migrations.go

```diff
@@ -45,7 +45,7 @@ func migrations(ctx *common.RenderContext) ([]runtime.Object, error) {
 					Spec: corev1.PodSpec{
 						Affinity:           common.NodeAffinity(cluster.AffinityLabelMeta),
 						RestartPolicy:      corev1.RestartPolicyNever,
-						ServiceAccountName: fmt.Sprintf("%s-migrations", Component),
+						ServiceAccountName: Component,
```

Comment:
Wanted to change the name of the job, and accidentally renamed the service account in https://github.com/gitpod-io/gitpod/pull/16066